packages/api-client/src/components/AddressBar/AddressBar.vue

```diff
@@ -109,6 +115,20 @@ const updateExampleUrlHandler = (url: string) => {
   if (!activeExample.value) return
   requestExampleMutators.edit(activeExample.value.uid, 'url', url)
 }
+
+/** Handle hotkeys */
+const handleHotKey = (event: HotKeyEvents) => {
+  if (event.focusAddressBar) {
+    addressBarRef.value?.focus()
+  }
+}
+
+onMounted(() => {
+  hotKeyBus.on(handleHotKey)
+})
+onBeforeUnmount(() => {
+  hotKeyBus.off(handleHotKey)
+})
```

Comment:
```suggestion
onMounted(() => hotKeyBus.on(handleHotKey))
onBeforeUnmount(() => hotKeyBus.off(handleHotKey))
```
when you only have one, it looks a bit cleaner 😁

Reply:
🍬 absolutely lets go!